pkg/alertmanager/multitenant_test.go

```diff
@@ -1980,6 +2001,27 @@ func TestSafeTemplateFilepath(t *testing.T) {
 	}
 }
 
+func TestIsFilePathInsideDirectory(t *testing.T) {
+	assert.False(t, isFilePathInsideDirectory("/", "/"))
+	assert.False(t, isFilePathInsideDirectory("/", ""))
+	assert.True(t, isFilePathInsideDirectory("/", "/test"))
```

Comment:
nit: could the cases expected to be `false` and the cases expected to be `true` be grouped? Maybe sub-tests?